### ISSUE TITLE
Variable/alternating row height

### ISSUE THREAD

**Issue by artemverbo** [miromannino/Justified-Gallery#102]
Hi there, thanks for a great library!L Love it! Was wondering if something like variable/alternating row heights can be possible, trying to achieve a similar grid look as here (if that particular listing will not be available, just open any other listing and to "Photos" tab) http://www.foxtons.co.uk/search?keyword_type=postcode&keyword_value=N1&property_id=942031&resource=thumbnails&search_form=keyword&search_type=SS&sold=1&submit_type=search The grid looks quite interesting in that case with lots of photos. I don't see anything like that in the options, but was wondering if somehow can achieve similar effect? 

Thank you!

**Comment by miromannino** [comment]
Yes it is possible, I will add the possibility to specify a function as height.

But the variable height in that gallery seems more about an error... And there are many error in that page, especially when one tries to resize it...

**Comment by th23x** [comment]
Cool, this is a really interesting option to make the gallery look even more dynamic :-)

Not absolutely sure about the option with a function - does that mean the function can determin the height per row rendered?
